### PR TITLE
Feature/publish public key

### PR DESCRIPTION
**Pull request by kchqwello**

### CODE REVIEW

File: modules/PowermeterBSM_WS36A/main/powermeterImpl.hpp [hunk 1]
```diff
@@ -68,6 +68,8 @@ class powermeterImpl : public powermeterImplBase {
     const Everest::PtrContainer<PowermeterBSM_WS36A>& mod;
     const Conf& config;
 
+    inline static const std::string PUBLIC_KEY_HEADER = "3059301306072A8648CE3D020106082A8648CE3D03010703420004";
```
Comment (golovasteek) [inline modules/PowermeterBSM_WS36A/main/powermeterImpl.hpp:71]: Add comment why it is here?

**Review by golovasteek — APPROVED** [review]
Looks good to me

File: modules/PowermeterBSM_WS36A/main/powermeterImpl.cpp [hunk 2]
```diff
@@ -89,12 +100,14 @@ void powermeterImpl::init() {
 
 void powermeterImpl::ready() {
     // Read the length of the register in bytes
+    EVLOG_info << "Reading public key";
```
Comment (SirVer) [inline modules/PowermeterBSM_WS36A/main/powermeterImpl.cpp:103]: I suggest leaving the logging here until this works reliably.

**Comment by SirVer** [comment]
looks good to me

**Review by golovasteek — APPROVED** [review]

**Pull request merged**